src/scss/foundations/typography/index.scss

```
@@ -10,12 +10,16 @@
 @use "text";
 @use "title";
 
+/// The font path
+/// @type String
+/// @access private
```

Domanda: mettiamo sempre uno spazio tra la descrizione e le varie keys?

```suggestion
/// The font path
///
/// @type String
/// @access private
```